WordPress/Classes/Services/EditorSettingsService.swift

```diff
@@ -19,11 +31,13 @@ import Foundation
         service.getEditorSettings(siteID, success: { (settings) in
             do {
                 try self.saveRemoteEditorSettings(settings, on: blog)
```

Comment:
I missed this in the previous PR, but `saveRemoteEditorSettings` seems like it's going to actually save the settings. Looking at the pattern in other services, I'd suggest `update(blog, remoteSettings: settings)`

Reply:
Pushed https://github.com/wordpress-mobile/WordPress-iOS/pull/12189/commits/5fef12b96b156fa4fc0103c08e430a4bd9ec7cff